src/components/BookList.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BookList = ({ books }) => {
+  const getBooks = () => {
+    const result = [];
+    for (let i = 0; i < books.length; i += 1) {
+      result.push(
+        <Book
+          key={books[i].id}
+          id={books[i].id}
+          title={books[i].title}
+          category={books[i].category}
+        />,
+      );
+    }
+    return result;
+  };
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>{getBooks()}</tbody>
+    </table>
+  );
+};
+
+BookList.propTypes = {
+  books: PropTypes.array,
+};
```

Comment:
- [ ] We can be more specific here an inform what is the type of each element in the books prop array:
```JS
BookList.propTypes = {
  books: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
  })),
};
```

- [ ] By doing that we can safely remove the eslint disable rule at the top of the file.